common/templates/layout.njk

```diff
@@ -1,23 +1,33 @@
-{% extends "../../govuk_modules/govuk_template/views/layouts/govuk_template.njk" %}
+{% from "components/footer/macro.njk" import govukFooter %}
 
-{% set homepage_url = '/' %}
+{% extends "template.njk" %}
 
-{% set logo_link_title = 'Go to the GOV.UK Homepage' %}
-
-{% block page_title %}GDS Node.js boilerplate{% endblock %}
+{% block pageTitle %}GDS Node.js boilerplate{% endblock %}
 
 {% block head %}
   {% include "common/templates/includes/head.njk" %}
 {% endblock %}
 
-{% block cookie_message %}
+{% block bodyStart %}
```

Comment:
If this is the cookie banner, you might want to consider the 'beforeContent' block instead.